### PR TITLE
Chronos : add scheduler for Autoformer

### PR DESCRIPTION
**Pull request by rnwang04**
## Description

add scheduler for Autoformer to fix https://github.com/intel-analytics/BigDL/issues/5096

### CODE REVIEW

File: python/chronos/src/bigdl/chronos/model/autoformer/Autoformer.py [hunk 1]
```diff
@@ -179,7 +179,9 @@ def predict_step(self, batch, batch_idx):
         return outputs
 
     def configure_optimizers(self):
-        return getattr(optim, self.optim)(self.parameters(), lr=self.lr)
+        optimizer = getattr(optim, self.optim)(self.parameters(), lr=self.lr)
+        scheduler = torch.optim.lr_scheduler.MultiStepLR(optimizer, milestones=[3,4,5,6,7,8,9,10], gamma=0.5, verbose=True)
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/model/autoformer/Autoformer.py:183]: I think we should allow users to set scheduler / milestones (I think this might be a better choice). Furthermore ,they may be allowed to disable scheduler.

Comment (rnwang04) [reply python/chronos/src/bigdl/chronos/model/autoformer/Autoformer.py]: done

File: python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py [hunk 2]
```diff
@@ -49,6 +49,7 @@ def __init__(self,
                  optimizer="Adam",
                  loss="mse",
                  lr=0.0001,
+                 lr_scheduler_milestones=[3, 4, 5, 6, 7, 8, 9, 10],
                  metrics=["mse"],
                  seed=None,
                  distributed=False,
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py:55]: add api doc (i.e. `:param lr_scheduler_milestones: xxxx`) for this parameter

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

File: python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py (hunk 2 above)
Comment (rnwang04) [reply python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py:55]: done
